pkg/querier/plan/plan.go

```diff
@@ -78,6 +80,24 @@ func (t QueryPlan) Equal(other QueryPlan) bool {
 	return bytes.Equal(left, right)
 }
 
+func (t QueryPlan) LineFilterExprs() []syntax.LineFilterExpr {
+	if t.AST == nil {
+		return nil
+	}
+	if t.lineFilterExprs == nil {
+		filters := syntax.ExtractLineFilterExprs(t.AST)
+		t.lineFilterExprs = &filters
```

Comment:
`t` is not a pointer

Reply:
here we are dereferencing `lineFilterExprs`

Reply:
Removed